07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -1555,6 +1555,50 @@ v) Override ControlSettings.json in ServerConfigMetaData.json as shown below:
 
 vi) Rerun the policy update or setup command (the same command you ran for the first-time setup).
 
+### How can we treat each public IP address as an individual resource?
+   Org policy admins can enable public IP address as an individual resource by making following org policy settings:
+
+i) Copy the ControlSettings.json from the AzSK installation to your org-policy folder.
+
+ii) Update the value to true in below tag under PublicIpAddress:
```

Comment:
Change to - Update the EnablePublicIpResource setting to true  - and in the snip below show it as true

Reply:
Made the required changes